tests/providers/jenkins/operators/test_jenkins_job_trigger.py

```diff
@@ -20,15 +20,23 @@
 
 import jenkins
 import mock
+from parameterized import parameterized
 
 from airflow.exceptions import AirflowException
 from airflow.providers.jenkins.hooks.jenkins import JenkinsHook
 from airflow.providers.jenkins.operators.jenkins_job_trigger import JenkinsJobTriggerOperator
 
+TEST_PARAMS = [
+    ({'a_param': 'blip', 'another_param': '42'},),
+    ('{"a_param": "blip", "another_param": "42"}',),
+    (['a_param', 'blip', 'another_param', '42'],),
+]
+
 
 class TestJenkinsOperator(unittest.TestCase):
+    @parameterized.expand(TEST_PARAMS)
     @unittest.skipIf(mock is None, 'mock package not present')
-    def test_execute(self):
+    def test_execute(self, the_parameters):
```

Comment:
```suggestion
    def test_execute(self, parameters):
```

Reply:
Good call. Just pushed this and a few other changes up 😅